tests/components/miele/snapshots/test_services.ambr

```diff
@@ -0,0 +1,125 @@
+# serializer version: 1
+# name: test_services_with_response
+  dict({
+    'programs': '''
+      [
+        {
+          "programId": 146,
+          "program": "QuickPowerWash",
+          "parameters": {}
+        },
+        {
+          "programId": 123,
+          "program": "Dark garments / Denim",
+          "parameters": {}
+        },
+        {
+          "programId": 190,
```

Comment:
currently this is one big string if I understand correctly. Can we make the response better and return proper JSON and use snake_case for fields. It would also be nice if we can have a few with parameters so the tests also test how that would look

Reply:
This is how it looks in the snapshot. The API returns a json (`return await res.json()`) and it looks perfectly nice in developer tools.
The received data is displayed unaltered. I am a bit reluctant to translate the received keys to snake case as it can be a maintenance problem. Miele don't flag when they make changes in the API. I can put a note in the docs on the need for changing programId to program_id if using yaml mode.

I will add a program entry for an oven with parameters and rename the fixture data file to program-test instead of washing-mashine.

Reply:
The thing with service responses is, is that we should not break them, because automations might depend on them. So if they don't flag when their API changes, we should add an extra translation layer to avoid breaking this in the future and have the schema in our control